src/EasyList/TodoServiceDB.cs

```diff
@@ -46,11 +46,13 @@ public void DisplayTodo(Todo todo)
         }
         public void DisplayAllTodo(TodoOrder todoOrder)
         {
+            int DisplayId = 0;
+
             var _todoList = _todoRepository.GetAllTodo(todoOrder)
-                            .Select(x => new { x.Id, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime() , DueDate = x.DueDate?.DateTime.ToLocalTime() });
+                            .Select(x => new { x.Id, DisplayId = DisplayId++, x.Label, x.Description, x.Priority, x.Status, CreateDate = x.CreatedDate.DateTime.ToLocalTime(), DueDate = x.DueDate?.DateTime.ToLocalTime() });
 
             ConsoleTableBuilder
-            .From(_todoList.ToList())
+            .From(_todoList.Select(x => new { DisplayId, x.Label, x.Description, x.Priority, x.Status, x.CreateDate, x.DueDate }).ToList())
```

Comment:
This select can be removed, we have already transformed the items now we just need to call .ToList

Reply:
then there will be Id and Display id.
how can i remove id from .can you give me any idea?

Reply:
I'm not sure how to since I'm not familiar with this specific package but you could check out the documentation and see if you find something for configuring the columns manually: https://github.com/minhhungit/ConsoleTableExt/

otherwise I'd suggest switching to this package https://github.com/khalidabuhakmeh/ConsoleTables

Reply:
I didn't find any solution to skip any column in console table. 
The table prints the given list value.
It has functionality to change able **title & column name.** 

So i have to work on list.is there any option **_without select_** to skip id?

Reply:
If we want to use a console table and keep the real id of the item then a select is probably the cleanest. Switching the console table package to the one i mentioned would be a simple solution